File: PRAK603/PRAK603-2210817310013-Ryan Muhammad Irfan.py
A = 0
x, y = map(int, input("").split())
matriks1 = []
matriks2 = []

if x == y:
    number1 = list(map(int, input("").split()))
    number2 = list(map(int, input("").split()))
    matriks1.append(number1[A: A + x])
    matriks2.append(number2[A: A + x])

    for i in range(0, x):
        print(matriks1[0][i]*matriks2[0][i], end=" ")

else:
    print("Jumlah tidak sama")

File: PRAK603/PRAK603-2210817310013-RyanMuhammadIrfan.c
#include <stdio.h>

int main()
{
    int a, x, y, i, j, number1[20], number2[20], hasil;

    scanf("%d %d", &x, &y);

    if (x == y)
    {
        for (i = 0; i < x; i++)
        {
            scanf("%d", &number1[i]);
        }
        for (i = 0; i < y; i++)
        {
            scanf("%d", &number2[i]);
        }

        for (i = 0; i < x; i++)
        {
            hasil = number1[i] * number2[i];
            printf("%d ", hasil);
        }
    }

    else
    {
        printf("Jumlah tidak sama");
    }
}